### PR TITLE
Added build against R 3.3.1.

### PR DESCRIPTION
**Pull request by vinjana**

### CODE REVIEW

**Comment by vinjana** [comment]
I like to have a build for Linux against R 3.3.1. Is this the way to do it?

**Comment by vinjana** [comment]
run_docker_build.sh contains a line "export CONDA_R=3.3.2". What does that exactly mean?

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+    - CONDA_R=3.3.1
```
Comment (ocefpaf) [inline .travis.yml:11]: The CIs configuration should not be changed manually. Only `conda-smithy` should be used for that.

With that said I am not sure `conda-forge` support other version of R besides `3.3.2`. @johanneskoester do you have any plans on this? Is it worth it?

Comment (bgruening) [reply .travis.yml:11]: @ocefpaf this pretty mich depends on how fast we can move on to R 3.4. If this takes longer we need either to build everything in bioconda against 3.3.2 or here a few packages against 3.3.1.

Comment (ocefpaf) [reply .travis.yml:11]: Thanks @bgruening. I am OK with either decision. (Actually 3.3.2 is fine for my limited R needs.)

Still... This PR is not how we should do it. Sorry @vinjana.

Comment (vinjana) [reply .travis.yml:11]: @ocefpaf O.k., I'm going to use conda smithy the next time.

For the publication of our workflow we'd only need 3.3.1. Going to 3.3.2 is not an easy decision if you need to reprocess terabytes of data and -- much worse -- check the results to ensure things are staying the same. Given that there are a number of 3.3.1 packages in bioconda I would then resort to adding the package version there.

Thanks.

**Pull request closed without merging**